charts/stable/stash/questions.yaml

```
@@ -60,6 +60,14 @@ questions:
             additional_attrs: true
             type: dict
             attrs:
+# Include{persistenceBasic}
+        - variable: blobs
+          label: "App Blobs Storage"
+          description: "Binary data for scene covers, performer images, etc"
```

lets stick with the template
```suggestion
          description: "Stores the Application Blobs."
```

---

Why? The following:

```text
label: "App Blobs Storage"
description: "Stores the Application Blobs."
```

just uselessly duplicates the information already contained within the label. I'd rather add useful, application-specific information from the applications docs:

https://docs.stashapp.cc/getting-started/installation/synology/#environment-tab